cucumber/authenticators_jwt/features/authn_jwt_token_schema.feature

```diff
@@ -0,0 +1,362 @@
+Feature: JWT Authenticator - Token Schema
```

Comment:
Do you think that there are too many tests.
Are we sure that all those use cases should be tested on integration level?
CC: @eladkug

Reply:
One another thought do you think we should separate tests of mapping and mandatory to different files and to add combined use case to the basic sanity file?

Reply:
But most of token schema tests are with both of them..

Reply:
So we add third file?

Reply:
@sashaCher Testing it on conjur its integration level its not E2E.
Running it on appliance act as E2E and we put there specific scenarios only.
Some of the test may insert to UT(Depends the time you save for each run) but its not a must.